tracex_project/db_results/templates/db_results_overview.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <title>Database Results</title>
     {% load static %}
-    <link rel="stylesheet" type="text/css" href="{% static '/tracex/css/style.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static '/tracex/css/styles.css' %}">
```

Comment:
why styles? instead of style?

Reply:
I first wanted to create multiple CSS documents for group-related styles since the original file was crowded and had no order. However, that would have meant importing multiple stylesheets in each HTML document. So, I just refactored the original file. In the process, I created a new file, `styles`, with a different name to compare them side by side.